code/game/atoms_movable.dm

```diff
@@ -74,11 +74,12 @@
 				AMob.grabbedby(src)
 			return TRUE
 		stop_pulling()
-	if(AM.pulledby)
+	/*if(length(AM.pulledby))
 		add_attack_logs(AM, AM.pulledby, "pulled from", ATKLOG_ALMOSTALL)
 		AM.pulledby.stop_pulling() //an object can't be pulled by two mobs at once.
+		*/ //May be removing this - Dragonkiller93
```

Comment:
Should remove this comment if you actually are removing it -- git will store it if you're worried about losing it.

That being said, you might want to restore the attack logs.

Reply:
I took it out because that no longer happens, an object can be pulled by two things at once, so there shouldn't be a pulled from.